packages/react-components/react-avatar/src/components/AvatarGroupItem/renderAvatarGroupItem.tsx

```diff
@@ -8,6 +8,10 @@ import type { AvatarGroupItemState, AvatarGroupItemSlots } from './AvatarGroupIt
 export const renderAvatarGroupItem_unstable = (state: AvatarGroupItemState) => {
   const { slots, slotProps } = getSlots<AvatarGroupItemSlots>(state);
 
-  // TODO Add additional slots in the appropriate place
-  return <slots.root {...slotProps.root} />;
+  return (
+    <slots.root {...slotProps.root}>
+      <slots.avatar {...slotProps.avatar} />
+      {state.isOverflowItem && slots.overflowLabel && <slots.overflowLabel {...slotProps.overflowLabel} />}
```

Comment:
(nit) does it work to remove the null check now that the slot is non-nullable?
```suggestion
      {state.isOverflowItem && <slots.overflowLabel {...slotProps.overflowLabel} />}
```

Reply:
It doesn't because there's still a ?, but I'll remove it as well